pkg/dogstatsd/server.go

```diff
@@ -341,6 +347,39 @@ func NewServer(demultiplexer aggregator.Demultiplexer, serverless bool) (*Server
 		}
 	}
 
+	// Creating a new logger
+	cfg := seelogCfg.NewSeelogConfig(
+		"dogstatsd",
+		"info",
+		"common",
+		"",
+		"%Date %Time | [DOGSTATSD_STATS] | [%LEV] %Msg ",
+		false)
+
+	// Configuring the log file path
+	logFile := config.Datadog.GetString("dogstatsd_log_file")
+	if logFile == "" {
+		logFile = constants.DefaultDogstatsDLogFile
```

Comment:
Could this be the default value in `BindEnvAndSetDefault`?

Reply:
I don't think it could be since we have different defaults for different platforms